compose/snippets/src/main/java/com/example/compose/snippets/layouts/MaterialLayoutSnippets.kt

```diff
@@ -375,7 +375,7 @@ fun BottomSheetDemo() {
     // [END android_compose_layout_material_bottom_sheet]
 
     // [START android_compose_layout_material_bottom_sheet2]
-    val sheetState = rememberSheetState()
+    val sheetState = rememberModalBottomSheetState()
```

Comment:
The CL for the corresponding DAC page still needs review from a Material SME, but `rememerModalBottomSheetState` should be the correct one. Thanks.